src/components/Table/TableBase.tsx

```diff
@@ -485,7 +484,6 @@ export class TableBase<T> extends React.Component<
 
 				<BaseTableWrapper>
 					<Base
-						{...props}
```

Comment:
Actually, I'm pretty sure that we need to wire in the className property, since we are using `styled(TableBase)` in `~src/components/Table/index.tsx` :thinking:

Reply:
Ah you're right, I'm seeing differences in styling between prod and local